packages/strategy-tests/src/transitions.rs

```diff
@@ -640,7 +640,7 @@ pub fn create_identities_state_transitions(
     rng: &mut StdRng,
     asset_lock_proofs: &mut Vec<(AssetLockProof, PrivateKey)>,
     platform_version: &PlatformVersion,
-) -> Result<Vec<(Identity, StateTransition)>, ProtocolError> {
+) -> Vec<(Identity, StateTransition)> {
```

Comment:
I think we should not error silently, I think a better approach would be to return a Vec<(Identity, Result<StateTransition, ProtocolError>)> we could also have another function as well create_identities_state_transitions_ignore_failures that filters out the failed results.

Reply:
👍🏻